src/components/players/detailsPlaceholder.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+import styles from './detailsPlaceholder.module.scss';
+
+const DetailsPlaceholder = () => {
+    return (
+        <div className={styles.playerDetailsContainer}>
+            <div className={styles.picture}/>
```

Comment:
Formatting:

`<div className={styles.picture}/>` should be `<div className={styles.picture} />`

Same for many instances in this file. Make sure to put space before `/>`